app/scripts/metamask-controller.js

```diff
@@ -1837,6 +1835,24 @@ export default class MetamaskController extends EventEmitter {
     return nonceLock.nextNonce
   }
 
+  async sendBackgroundMetaMetrics ({ action, name, customVariables } = {}) {
+
+    if (!action || !name) {
+      throw new Error('Must provide action and name.')
+    }
+
+    if (this.preferencesController.getParticipateInMetaMetrics()) {
```

Comment:
I think this is redundant - we check that the user has opted in here as well: https://github.com/MetaMask/metamask-extension/pull/9144/files#diff-eb1994c6a9718a73c4d60038f5e23b95R6

Reply:
Fixed in: 14f6452